protocols/JabberG/src/jabber_opt.cpp

```diff
@@ -687,7 +685,7 @@ class CDlgOptAccount : public CJabberDlgBase
 	{
 		m_gotservers = node != nullptr;
 
-		wchar_t *server = m_cbServer.GetText();
+		wchar_t *server = m_c.GetText();

```

Comment:
That does not even compile. Once again, please compile and test your changes locally, otherwilse this PR will get closed.